src/test/java/org/apache/commons/io/FileUtilsCopyDirectoryToDirectoryTestCase.java

```diff
@@ -73,6 +83,48 @@ public void copyDirectoryToDirectoryThrowsNullPointerExceptionWithCorrectMessage
         assertExceptionTypeAndMessage(srcDir, destDir, NullPointerException.class, "destinationDir");
     }
 
+
+    @Test
+    public void copyFileWrongPermissions() throws IOException {
+        
+    	 
+        final File destDir = createTemporaryFolderIn(null);
+        final  Path srcFile = Files.createTempFile("tmp-output", ".xml");
+        final Path path = Paths.get(destDir.getAbsolutePath(), "newFile.xml");
+
+        try {
+            FileUtils.copyFile(srcFile.toFile(), path.toFile());
+        } catch (IllegalArgumentException iae) {
+        	iae.printStackTrace();
+        }
+        
+        assertTrue(Files.getPosixFilePermissions(path).contains(PosixFilePermission.OTHERS_READ), Files.getPosixFilePermissions(path).toString());
+
+    }
+    
+    
+    
+    private File createTemporaryFolderIn(File parentFolder) throws IOException {
```

Comment:
What is this all about? Since we are testing `FileUtils.copyFile()`, why do we need an extra level of temp folder beyond what the existing `temporaryFolder` provide?